qiskit/circuit/library/standard_gates/u3.py

```diff
@@ -27,7 +27,7 @@ class U3Gate(Gate):
 
     .. math::
         U3(\theta, \phi, \lambda) =
-            RZ(\phi - \pi/2) RX(\pi/2) RZ(\pi - \theta) RX(\pi/2) RZ(\lambda - \pi/2)
+            RZ(\phi) RX(\pi/2) RZ(\pi - \theta) RX(\pi/2) RZ(\lambda + \pi)
```

Comment:
Thanks, this is a mistake. But can you use the decomposition that Qiskit uses for this, for consistency?
```suggestion
            RZ(\phi + \pi) RX(\pi/2) RZ(\theta + \pi) RX(\pi/2) RZ(\lambda)
```
This is how you get Qiskit's decomposition:
```py
from numpy import pi
from qiskit.circuit.library import RZGate, RXGate, U3Gate
theta = 0.1
phi = 0.2
lam = 0.3

u3 = Operator(U3Gate(theta, phi, lam))

from qiskit.quantum_info.synthesis import OneQubitEulerDecomposer
print(OneQubitEulerDecomposer('U1X')(u3))

     ┌─────────┐┌──────────┐┌────────────┐┌──────────┐┌────────────┐
q_0: ┤ U1(0.3) ├┤ RX(pi/2) ├┤ U1(3.2416) ├┤ RX(pi/2) ├┤ U1(3.3416) ├
     └─────────┘└──────────┘└────────────┘└──────────┘└────────────┘


```

Reply:
Sure - I noticed that was an alernative answer but I tried to keep the pi - theta in the middle. I found the same mistake in the U2 docs (with 2 possible answers) so I'll check if my PR for that one is consistent, too. Does this mean this is how they are implemented in real IBMQ backends as well? (In terms of picking phase offsets for the drive pulses). And do I need to do anything to approve the change (I'm very new to open source)? Thanks for your patience.

Reply:
Each backend reports how it implements gates with pulses/phase shifts the `backend.defaults()` datastructure that is returned from them (for pulse enabled backends). There's no "correct" way as both are valid here, I just thought if someone uses Qiskit's decomposer it would be nice to be consistent.

I can't see the merge button here for some reason. So maybe you can just change your branch and push again.

Thanks again for catching this error and fixing.

Reply:
Thanks, pushed again and now at #4772. 
Realised what I did wrong previously (still getting used to Github) - thanks for your patience and I'm closing this now.